make/Makefile.iso

```diff
@@ -82,7 +82,7 @@ else
 endif
 
 .PHONY: iso
-iso: $(LUET) $(YQ) $(MAKEISO) $(MKSQUASHFS)
+iso: $(LUET) yq $(MAKEISO) $(MKSQUASHFS)
```

Comment:
I guess `$(MAKEISO)` here should be substituted to `makeiso` right?

Reply:
Good catch!